arm-sql/2015-05-01/swagger/engineAuditRecords.json

```diff
@@ -0,0 +1,225 @@
+            "type": "object",
+            "properties": {
+                "error": {
+                    "$ref": "#/definitions/ArmErrorResponseExtendedErrorInfo"
+                }
+            }
+        },
+        "ArmErrorResponseExtendedErrorInfo": {
+            "type": "object",
+            "properties": {
+                "details": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ArmErrorResponseErrorDetail"
+                    }
+                },
+                "innerError": {
+                    "$ref": "#/definitions/Object"
+                },
+                "code": {
+                    "type": "string"
+                },
+                "message": {
+                    "type": "string"
+                },
+                "target": {
+                    "type": "string"
+                }
+            }
+        },
+        "ArmErrorResponseErrorDetail": {
+            "type": "object",
+            "properties": {
+                "code": {
+                    "type": "string"
+                },
+                "message": {
+                    "type": "string"
+                },
+                "target": {
+                    "type": "string"
+                }
+            }
+        },
+        "EngineAuditRecordsFilterParameters": {
+            "type": "object",
+            "properties": {
+                "eventTime": {
+                    "format": "date-time",
+                    "type": "string"
+                },
+                "showSystemLogs": {
+                    "type": "boolean"
+                },
+                "showServerRecords": {
+                    "type": "boolean"
+                }
+            }
+        },
+        "EngineAuditRecordResourceList": {
+            "type": "object",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/EngineAuditRecordResource"
+                    }
+                },
+                "nextLink": {
+                    "type": "string"
+                }
+            }
+        },
+        "EngineAuditRecordResource": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "string"
+                },
+                "name": {
+                    "type": "string"
+                },
+                "type": {
+                    "type": "string"
+                },
+                "location": {
+                    "type": "string"
+                },
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/EngineAuditRecordResourceProperties"
+                }
+            }
+        },
+        "EngineAuditRecordResourceProperties": {
+            "type": "object",
+            "properties": {
```

Comment:
- missing description in all the properties of this object.
- Are there any required properties?
- Are there any readonly properties ?